docs/pages/attributesJSON.rst

```diff
@@ -29,7 +29,7 @@ Below are the supported JSON tags for Physics Manager Attributes templates, and
 
 "physics_simulator"
 	- string
-	- What physics engine should be used for dynamics simulation.  Currently supports "bullet" for Bullet physics simulation, and "none", meaning kinematic motion is to be used.
+	- What physics engine should be used for dynamics simulation. Currently supports "bullet" for Bullet physics simulation, and "none", meaning kinematic motion is to be used.
```

Comment:
Can we add a sentence here saying what the implications of using kinematic motion are? Collisions still happen, but you can't apply forces.